jenkins/test-history/buckets.json

```diff
@@ -0,0 +1,11 @@
+{
+    "gs://kubernetes-jenkins/logs/": {
+        "prefix": ""
+    },
+    "gs://kube_azure_log": {
+        "prefix": "azure$"
+    },
+    "gs://rktnetes-jenkins/logs/": {
+        "prefix": "rktnetes$"
```

Comment:
It is a separator.  @rmmh used that originally, and I carried it forward.

I've been wondering what kind of naming scheme we want for federated tests, and what should be the semantics of the prefix.  Do we want to establish a uniform naming scheme?

Consider these existing test suites: kubernetes-e2e-gce, kubernetes-e2e-aws.  The cloud provider is embedded in the name.  So, one logical name for the Azure tests would be kubernetes-e2e-azure.  The prefix "azure$" becomes redundant.

For the rknetes tests, I'm less sure what to suggest as rkt is a container runtime, which is independent of the could provider.  Should the name look like "kubernetes-e2e-gce-rkt"?